docs/engineering/tooling/readme.mdx

```diff
@@ -0,0 +1,8 @@
+# fhircore-tooling
+
+A repo to hold our FHIR content and configuration creation tools and scripts.
+
+- [cleaner](https://github.com/onaio/fhircore-tooling/tree/main/cleaner)
+- [efsity](https://github.com/onaio/fhircore-tooling/tree/main/efsity)
+- [importer](https://github.com/onaio/fhircore-tooling/tree/main/importer)
+- [uploader](https://github.com/onaio/fhircore-tooling/tree/main/uploader)
```

Comment:
I think all these links should be removed, we can add brief descriptions of each tool and link to the docs sub-page